buildSrc/src/main/java/org/opensearch/gradle/DistributionDownloadPlugin.java

```diff
@@ -174,6 +176,19 @@ private static void setupDownloadServiceRepo(Project project) {
             return;
         }
         Object customDistributionUrl = project.findProperty("customDistributionUrl");
+        Object bundleDownload = project.findProperty("bundleDownload");
+        boolean bundleBoolean = bundleDownload != null && Boolean.parseBoolean(bundleDownload.toString());
+        if (bundleBoolean) {
+            addIvyRepo(project, DOWNLOAD_REPO_NAME, "https://ci.opensearch.org", FAKE_IVY_GROUP, "/ci/dbc" + BUNDLE_PATTERN_LAYOUT);
+            addIvyRepo(
+                project,
+                SNAPSHOT_REPO_NAME,
+                "https://ci.opensearch.org",
+                FAKE_SNAPSHOT_IVY_GROUP,
+                "/ci/dbc" + BUNDLE_PATTERN_LAYOUT
```

Comment:
Why not just add `ci/dbc` to the pattern?

Reply:
Just updated to add it to the pattern.